week-2/implement/contains.js

```diff
@@ -1,3 +1,8 @@
-function contains() {}
+function contains(obj,key) {
```

Comment:
obj: should not be null and includes an array
key: should be a string value
If obj is null, return false.
If obj is not an object or an array, return false.
If key is not a string, return false.
If obj is no
contains([1, 2], 1) it returns false because the function takes key as string 
in this case contains([1, 2], '1') it returns true

Reply:
`contains([1, 2], '1')` should return true?
It doesn't make any sense, ahah)
1 is not equal to '1'.
I would understand `contains(['1', '2'], '1')` returning true.
Is it how the task is given? Or it's your interpretation?

Reply:
Hi Sergey, 
Yes you are true It should return false for array input I change the code and I think Its good now 
function contains(obj,key) {
  if(typeof obj !== "object" || obj === null || Array.isArray(obj)){
    return false;
  } 
  return obj.hasOwnProperty(key);
}
 test('should return false as an array input', () => {
    expect(contains([1, 2], '1')).toBe(false);
  });